tests/test_bgp_af.rb

```diff
@@ -106,6 +110,41 @@ def test_collection_not_empty
     end
   end
 
+  def config_ios_xr_dependencies(asn, vrf='red', casty='unicast')
+    # These dependencies are required on ios xr
+
+    # router bgp 55
```

Comment:
Nit. I think you can condense this example in the comments? 

And there's a couple of unnecessary comments in there like 'Configure'.

Also not sure what calling the variable 'casty' represents?

Is the loopback needed?